docs/documentation/stories/global-lib.md

```diff
@@ -12,7 +12,7 @@ First install Bootstrap from `npm`:
 ```bash
 npm install jquery --save
 npm install popper.js --save
```

Comment:
It's not something you added, but can we add that neither `jquery` nor `popper.js` are needed if you pretend to use only CSS? With the current docs people tend to feel that they have to install them obligatory.

Reply:
I totally agree with you. I will update my PR accordingly.

Reply:
I updated the PR. Tell me what you think about the changes.